Resources/translations/messages.de.xliff

```diff
@@ -18,6 +18,22 @@
         <source>notification.spam_complaint_received</source>
         <target>Spambeschwerde erhalten</target>
       </trans-unit>
+      <trans-unit id="egx5j005ff11858e5v62i72fb99337ff9e7925f5" resname="notification.blacklist_received">
+        <source state="new">notification.blacklist_received</source>
```

Comment:
for all translation-files: afaik the `state="new"` attribute should be on the <target> element and not the <source> element. See http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#state